src/web/arr/trove/chart.arr

```diff
@@ -443,6 +451,8 @@ data ChartWindow:
     constr: {(): box-plot-chart-window},
     x-axis: x-axis-method,
     y-axis: y-axis-method,
+    min: min-method,
```

Comment:
This will not work, because `default-box-plot-chart-window-object` doesn't have `max` and `min` fields.

More generally, are `max` and `min` properties of the chart window, or properties of data?

Reply:
They're the settings of the chartwindow - but I thought any settings there had to apply to every chart type?

Reply:
Oh, ok. Sorry about "an error will be thrown". That's not true. It's true however that this will not work as intended.

Reply:
In that case, https://github.com/brownplt/code.pyret.org/commit/a00cefc5bf46a8e0660d887504a9b7e98e460415 should be reverted, and

```
  min :: Option<Number>,
  max :: Option<Number>
```

and 

```
  min: none,
  max: none
```

should be added to `BoxChartWindowObject` and `default-box-plot-chart-window-object` respectively. The JS code is also needed to be modified to retrieve the information from `globalOptions` instead of `rawData`.

As for this PR, 

```
    method max(self :: DataSeries, max :: Number) -> Option<Number>:
      self.constr()(self.obj.{max: some(max)}
    end
```

should be removed entirely. Adding `min: min-method,` is the correct way to do it.

Reply:
@sorawee I'm with you up until the last part. Are you saying I should rewrite:

```
    method min(self :: DataSeries, min :: Number):
      self.constr()(self.obj.{min: some(min)})
    end,
    method max(self :: DataSeries, max :: Number):
      self.constr()(self.obj.{max: some(max)})
    end
```

as

```
min: min-method,
max: max-method
```
?

Reply:
Not really. I meant:

```
    method min(self :: DataSeries, min :: Number):
      self.constr()(self.obj.{min: some(min)})
    end,
    method max(self :: DataSeries, max :: Number):
      self.constr()(self.obj.{max: some(max)})
    end
```

should be removed entirely, since you want `min` and `max` to operate on `ChartWindow`, not `DataSeries`. And

```
min: min-method,
max: max-method
```

which operates on `ChartWindow` in the current PR is correct already.